core/src/main/java/org/kiwix/kiwixmobile/core/page/viewmodel/PageState.kt

```diff
@@ -26,7 +26,7 @@ abstract class PageState<T : Page> {
   val isInSelectionState: Boolean by lazy { pageItems.any(Page::isSelected) }
   protected val filteredPageItems: List<T> by lazy {
     pageItems.filter { showAll || it.zimId == currentZimId }
-      .filter { it.title.contains(searchTerm, true) }
+      .filter { it.title?.contains(searchTerm, true) ?: false }
```

Comment:
Here we don't need to set explicitly set to false if the `pageItem` s `title` are non-nullable.

Reply:
done